tutorials/e-commerce-store-codelab/step-03-configuring-drizzle.md

```diff
@@ -0,0 +1,457 @@
+---
+description: Configuring Drizzle as the ORM for a e-commerce app built with Remix and Turso.
+keywords:
+  - turso
+  - codelab
+  - drizzle
+  - remix
+  - e-commerce
+---
+
+# Step 3: Configuring Drizzle
+
+As prescribed earlier we are going to use Drizzle to handle our database schema
+and build queries in our app. Run the following commands to install its
+dependencies in the Remix project.
+
+```sh
+npm install drizzle-orm @libsql/client
+npm install -D drizzle-kit dotenv tsx
+```
+
+_**Note:** The `dotenv` npm package added above is going to help us use
+environment variables stored within the `.env` file within our Drizzle code.
+We’ll use the `tsx` package to execute TypeScript files mainly within
+`package.json` scripts._
+
+Then, create a `/drizzle` directory at the project’s root.
+
+We are going to place our app’s database schema and store Drizzle migrations
+inside this directory.
+
+Create a `.env` file at the project’s root and add the database credentials
+obtained in the previous section.
+
+```sh
+# /.env
+
+TURSO_DB_URL=<OBTAINED-TURSO-DB-URL>
+TURSO_BD_AUTH_TOKEN=<OBTAINED-TURSO-DB-TOKEN>
+```
+
+_**Note:** Do not stage `.env` files to your version control systems._
+
+Our e-commerce database is going to have the following tables.
+
+- Categories - Stores the products’ categories
+- Products - Stores the website’s products
+- Users - Stores customer information
+- Passwords - Stores customer passwords
+- Cart Items - Stores items added to cart by customers
+- Orders - Stores the order information created by customers
+- Order Items - Stores the products belonging to orders
```

Comment:
These don't look like the actual table names. Probably better to use those instead.